recommendation/microsoft.owin.security.json

```diff
@@ -0,0 +1,752 @@
+{
+  "Name": "Microsoft.Owin.Security",
+  "Version": "1.0.0",
+  "Packages": [
+    {
+      "Name": "Microsoft.Owin.Security",
+      "Type": "Nuget"
+    }
+  ],
+  "Recommendations": [
+    {
+      "Type": "Namespace",
+      "Name": "Microsoft.Owin.Security",
+      "Value": "Microsoft.Owin.Security",
+      "KeyType": "Name",
+      "ContainingType": "",
+      "RecommendedActions": [
+        {
+          "Source": "Amazon",
+          "Preferred": "Yes",
+          "TargetFrameworks": [
+            {
+              "Name": "netcoreapp3.1",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            }
```

Comment:
do we also want to support `net5.0` for this action?

Reply:
Line 83-85 covers what actions are related to .net5, so no worries it is also covered.